guides/v2.3/graphql/tutorials/checkout/checkout-add-product-to-cart.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: tutorial
+group: graphql
+title: Step 3. Add product to cart
+subtitle: GraphQL checkout tutorial
+level3_subgroup: graphql-checkout
+return_to:
+  title: GraphQL Overview
+  url: graphql/index.html
+menu_order: 30
+functional_areas:
+  - Integration
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+GraphQL supports [two types of product]({{ page.baseurl }}/graphql/reference/product-interface-implementations.html) which can be added into shopping cart:
+ 
+ * simple product
+ * virtual product
+
+{:.bs-callout .bs-callout-info}
```

Comment:
Use the same note that's in Step 4 (set shipping address) and place it immediately before the mutation.

Reply:
**done**